src/views/Applications/ApplicationForm.vue

```diff
@@ -213,22 +232,25 @@ import usePortalApi from '@/hooks/usePortalApi'
 import cleanupEmptyFields from '@/helpers/cleanupEmptyFields'
 import useToaster from '@/composables/useToaster'
 import { useI18nStore, useAppStore } from '@/stores'
-import { CreateApplicationPayload, UpdateApplicationPayload } from '@kong/sdk-portal-js'
+import { CreateApplicationPayload, ListAuthStrategiesItem } from '@kong/sdk-portal-js'
 import { FeatureFlags } from '@/constants/feature-flags'
 import useLDFeatureFlag from '@/hooks/useLDFeatureFlag'
+import { fetchAll } from '@/helpers/fetchAll'
 
 export default defineComponent({
   name: 'ApplicationForm',
   components: { PageTitle, CopyButton },
 
   setup () {
-    function makeDefaultFormData (isDcr:boolean): UpdateApplicationPayload {
+    function makeDefaultFormData (isDcr:boolean): CreateApplicationPayload {
       const returnObject = {
         name: '',
         description: '',
         redirect_uri: '',
-        reference_id: ''
+        reference_id: '',
+        auth_strategy_id: ''
```

Comment:
Is `auth_strategy_id` ignored if using v0 API?

Reply:
yes as we don't fetch the auth strategies or display the select for it if the featureflag is disabled, empty fields are ignored